telephone/urls.py

```diff
@@ -30,7 +28,7 @@
     # inspector views
     url(r'^(?P<pk>\d+)/inspect/$', inspect_views.InspectView.as_view(),
         name='inspect'),
-    url(r'^(?P<pk>\d+)/inspect/api/', include(router.urls)),
+    url(r'inspect/api/', include('inspector.urls')),
```

Comment:
Please note, that I've changed urls a bit. That shouldn't affect the code outside inspector app, but I want to warn you in case I overlooked something.